R/helper.functions.R

```diff
@@ -81,14 +77,16 @@ get.drho_dz <- function(wtr, depths){
 }
 
 
-#@title Find and drop the datetime column from the datatable
-#
-#description Liberally looks for a datetime column and drops it, 
-#returning a data.frame with only water temperature. Errors if datetime column is 
-#ambiguous. Warns if there is no match.
-#
-#@return A data.frame with only the data, after datetime has been dropped
-#
+#' @title Find and drop the datetime column from the datatable

```

Comment:
`@title` `@description` used here, not elsewhere. Please include in other functions.

Reply:
If you omit the title and description tag in first two entries they are automatically converted to title and description. But you'd prefer to have those tags explicitly stated?